interfaces/vrep/DQ_VrepInterface.m

```diff
@@ -461,12 +590,28 @@ function set_joint_positions(obj,handles,thetas,opmode)
             end
         end
         
-        %% Set Joint Target Positions
-        function set_joint_target_positions(obj,handles,thetas,opmode)
-            %% Set the joint target positions of a robot in V-REP. For joints that are in 'Force/Torque Mode' in V-REP
-            %%  >> joint_names = {'redundantRob_joint1','redundantRob_joint2','redundantRob_joint3','redundantRob_joint4','redundantRob_joint5','redundantRob_joint6','redundantRob_joint7'};
-            %%  >> vi.set_joint_target_positions(joint_names,[0 pi/2 0 pi/2 0 pi/2 0]);     
+        
+        function set_joint_target_positions(obj,jointnames,joint_target_positions,opmode)
+            % This method sets the joint target positions of a robot in the server. 
```

Comment:
Same comments as before. Here, it is worth pointing to some references about dynamic scenes and how to set up a joint in dynamic mode with position control mode (e.g., CoppeliaSim documentation, although in the future, we might point to some of our tutorials).